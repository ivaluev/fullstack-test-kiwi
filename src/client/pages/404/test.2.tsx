import React from "react"
import NotFound, { Props } from "."

import { shallow, ShallowWrapper } from "enzyme"

function setUp(props: Props = {}) {
  const wrapper = shallow(<NotFound {...props}/>)
  return wrapper
}

describe("NotFound Component (ENZ)", () => {

  let component: ShallowWrapper
  beforeEach(() => {
    component = setUp()
  })

  it("Should render without errors", () => {
    const txt = component.find(`[data-testid="NotFound"]`)
    expect(txt.length).toBe(1)
  })

  it("Should match snapshot", () => {
    expect(component).toMatchSnapshot()
  })
})