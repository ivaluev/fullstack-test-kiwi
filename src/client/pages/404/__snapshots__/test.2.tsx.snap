// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component (ENZ) Should match snapshot 1`] = `
<div>
  <span
    data-testid="NotFound"
  >
    Not Found
  </span>
</div>
`;
